index.js

```diff
@@ -20,6 +20,12 @@ function increaseSpecifityOfRule(rule, opts) {
 		) {
 			return selector + opts.stackableRoot.repeat(opts.repeat);
 		}
+		
+		// Avoid prefix for :exports since :export block defines the symbols 
+		// that are going to be exported to the consumer
+		if (selector.indexOf(":export") !== -1) {
+		    return selector;
+		}
```

Comment:
Seems fine to me. But let's add a test for this case so we don't regress on this in the future.

Reply:
Done, test added